content/en/docs/CommunityLeaders/EventHandbooks/Docker101/gettingstarted/_index.md

```diff
@@ -82,6 +82,71 @@ For more examples and ideas, visit:
  https://docs.docker.com/get-started/
 ```
 
+###  If You Are Facing This Error 
+  ```
+$ docker run hello-world
+```
+```
+docker: Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/create": dial unix /var/run/docker.sock: connect: permission denied.
+See 'docker run --help'.
+```
+
+
+## Add "sudo" Before Run Any Docker Command 
```

Comment:
While both the solutions you have provided will work, but these are not recommended. You may want to check out this document on the Docker website: [https://docs.docker.com/engine/install/linux-postinstall/](https://docs.docker.com/engine/install/linux-postinstall/).